algebird-test/src/test/scala/com/twitter/algebird/QTreeTest.scala

```diff
@@ -69,6 +69,15 @@ class QTreeTest extends WordSpec with Matchers {
     list.filter{ _ >= from }.filter{ _ < to }.sum
```

Comment:
this line wasn't touched in my diff, nor was it called... I can change it if you want but it seems out of scope